src/server/server.ts

```diff
@@ -172,22 +170,24 @@ namespace ts.server {
         }
 
         perftrc(s: string) {
-            this.msg(s, Msg.Perf);

```

Comment:
Could we instead turn `Msg` into a string enum? That would allow us to continue to find all references if necessary.